src/Microsoft.VisualStudio.SlnGen/ProjectFileExtensions.cs

```diff
@@ -77,7 +77,12 @@ internal static class ProjectFileExtensions
         /// <summary>
         /// SQL Server database projects (.sqlproj).
         /// </summary>
-        public const string SqlServerDb = ".sqlproj";
+        public const string SqlServerDbLegacy = ".sqlproj";
```

Comment:
This seems it could be condensed to use:
`public const string SqlServerDb = ".sqlproj"`

Which can then be referenced in `SlnProject`. Is there a reason for having a separate const variable corresponding to the same string out of curiosity?

Reply:
Makes sense, I have chnaged it now.